docs/team/jellybeano.md

```diff
@@ -0,0 +1,120 @@
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to add unique user tags.
+    * What it does: Allow the users to add tags to flashcards that are separate from the default and immutable tags that the application provides.
+    * Justification: This feature improves the product significantly because the application provides functions that allow for viewing and sorting flashcards by their tags, which complements the Learn mode of the application greatly.
+    * Highlights: This enhancement affected one of the key modes, Learn Mode, of the application greatly and became one of the main features of that mode, along with viewing all flashcards.
+      Tagging also affected most other sections of the application and thus required deeper insight into how the entire application functioned.
+    * Credits: The code took some inspiration from the original Address Book that the project was morphed from.
+
+* **New Feature**: Added the deleteTag command that tied in with the above command. Users could delete tags, but not default ones set by the application.
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s2.github.io/tp-dashboard/#breakdown=true&search=jellybeano)
+
+* **Project management**:
+    * Coordinated and led project meetings.
+    * Assisted in managing milestones for the team.
+
+* **Enhancements to existing features**:
+    * Updated the GUI color scheme and added visual improvements (Pull request [\#60](https://github.com/AY2021S2-CS2103T-T13-1/tp/pull/60))
+
+* **Documentation**:
+    * User Guide:
+        * Added documentation for the features `learn`, `tag` and `deleteTag` (Pull requests [\#96](https://github.com/AY2021S2-CS2103T-T13-1/tp/pull/96), [\#111](https://github.com/AY2021S2-CS2103T-T13-1/tp/pull/111))
+        * Update the diagrams to match the upstream repo after they were changed (Pull request [\#111](https://github.com/AY2021S2-CS2103T-T13-1/tp/pull/111))
+    * Developer Guide:
+        * Added implementation details of the `learn` feature and updated various UML diagrams. (Pull requests [\#81](https://github.com/AY2021S2-CS2103T-T13-1/tp/pull/81), [\#96](https://github.com/AY2021S2-CS2103T-T13-1/tp/pull/96))
+        * Added implementation details of the tagging feature, along with relevant diagrams. (Pull request [\#81](https://github.com/AY2021S2-CS2103T-T13-1/tp/pull/81))
+
+* **Team-Based Tasks**:
+    * Assisted with refactoring and morphing Address Book into the current application. (Pull request [\#22](https://github.com/AY2021S2-CS2103T-T13-1/tp/pull/22))
+    * Changed the icon for the application. (Pull request [\#179](https://github.com/AY2021S2-CS2103T-T13-1/tp/pull/179))
+    * Helped maintain the issue tracker for the team repository. 
+    
+## Contributions to the User Guide
+
```

Comment:
Wait were we supposed to paste our sections of the UG here?

Reply:
oh i was assuming that when they gave 1 page to paste optional contributions to user guide this was what they meant but now that i think about it